com.unity.ml-agents/Runtime/Agent.deprecated.cs

```diff
@@ -0,0 +1,38 @@
+using System;
```

Comment:
Holder for deprecated methods.  Commented them out, for now, otherwise, we can't compile.

Reply:
> we can't compile

Is that because of the warnings-as-errors setting? I thought the obsolete warning was excluded.

Reply:
warnings-as-errors

Reply:
Hmm... we can save this for later, but should try to get it figured out.